### ISSUE TITLE
Refresh Buttons Missing Voiceover Accessibility

### ISSUE THREAD

**Issue by ghost** [brave/brave-ios#2184]
### Description: 
voiceover cannot find any button labeled as refresh

 
### Steps to Reproduce 
  1. open any page
  2. use voiceover to explore the ui
  3. cannot find the refresh button

**Actual result:** 

**Expected result:**
expected that I can find a refresh button to refresh the page

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
everytime

**Brave Version:** 
1.14.2 版 (19.12.12.17)

**Device details:** 
iphone 8 ios 13.3
